www/src/components/PageLayout.tsx

```diff
@@ -46,13 +49,13 @@ const Layout = ({
         siteTitle={data.site.siteMetadata?.title || 'Title'}
         showMinimizedTitle={showMinimizedTitle}
       />
-      <Settings />
+      <Settings showMinimizedTitle={showMinimizedTitle} />
       {isMdx ? (
         <Container fluid>
```

Comment:
[curious/clarification] If the setting is controlling container width, should the setting be controlling the `size` prop for the `Container` component?

For example: `<Container size={settings.containerWidth} />`

Reply:
Of course, I added

Reply:
![image](https://user-images.githubusercontent.com/93188219/202701553-8d816140-2cdd-4366-bd9f-d113f6ceeb18.png)